main.py

```diff
@@ -25,6 +25,17 @@
 
 API_URL="https://api.github.com"
 
+def test_rate():
+    url = "https://api.github.com/rate_limit"
```

Comment:
This should use `API_URL`?

Reply:
I was just about to push a commit fixing that! lol